antora.yml

```
@@ -96,3 +95,8 @@ asciidoc:
     project-context: che
     repo-path: ""
     theia-endpoint-image: eclipse/che-theia-endpoint-runtime:next
+    prod-migration-id-short: che
```

`prod-migration-id-short` is the short ID of the product before the update right? So why not:
```suggestion
    pre-migration-prod-id-short: che
```
and the same for other variables

---

Agree prod-migration -> pre-migration-prod